openssl-tracer/ssl_client_server/client.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should we actually be checking these certs in? Instead, should we just give them steps for how to generate their own certs? It'll be annoying to renew these certs when they expire later, for example.

Reply:
Yeah, I'm in the process of giving instructions instead.